cartography/intel/aws/ec2/instances.py

```diff
@@ -76,7 +76,8 @@ def load_ec2_instance_network_interfaces_tx(
             nic.lastupdated = $update_tag
             
             WITH nic, interface
-            MERGE (instance:EC2Instance{instanceid: interface.InstanceId})-[r:NETWORK_INTERFACE]->(nic)
+            MATCH (instance:EC2Instance{instanceid: interface.InstanceId})
+            MERGE (instance)-[r:NETWORK_INTERFACE]->(nic)
```

Comment:
revert back it

Reply:
This `MERGE (instance:EC2Instance{instanceid: interface.InstanceId})-[r:NETWORK_INTERFACE]->(nic)`  query is creating another instance with only instanceid field set. But we want to create relationship with existing instance.
That's why I have first matched the instance and then merging the relationship with that instance.
If I revert it back, some of the existing tests will not work.